platform/linux-generic/odp_crypto.c

```diff
@@ -1060,6 +1081,41 @@ int odp_crypto_term_global(void)
 	return rc;
 }
 
+int odp_crypto_init_local(void)
```

Comment:
`_odp_crypto_init_local`

Reply:
I don't see the need to do that at this point as these are internal routines anyway. That type of restructure might be something to consider for Caterpillar.

Reply:
Then I would prefer to use names w/o underscore for now and to change all of them later.

Reply:
That makes sense to me.

Reply:
I prefer the rule that new code goes in in clean way. Even if rest of the code used old rules.

Reply:
Then I'd follow "Maxim's rule" and do the renames as a separate follow-on PR.